Dockerfile

```diff
@@ -7,18 +7,17 @@ WORKDIR /stage
 RUN conda create -n runenv --yes python=3.5
 ENV PYTHONHASHSEED 2157
 
-# Copying and installing required python packages.
-COPY requirements.txt requirements.txt
-# requirements.txt uses the CPU version of tensorflow, for Travis CI; we want the GPU version, so
-# we use sed to change it before installing the requirements.
-RUN sed -ie 's/^tensorflow/tensorflow-gpu/g' requirements.txt
-RUN /bin/bash -c "source activate runenv && pip install -r requirements.txt"
-RUN /bin/bash -c "source activate runenv && python -m nltk.downloader punkt"
-RUN /bin/bash -c "source activate runenv && python -m spacy.en.download all"
-
-# Copying the source code.
+# Copying the source code and pip requirements.
 COPY scripts/ scripts/
 COPY deep_qa/ deep_qa/
+COPY requirements.txt requirements.txt
```

Comment:
Good point.  I added a dependency on `scripts` for installing pip requirements however.  Which of the following would you prefer:

1.  Move `COPY script/ script/` earlier in the Dockerfile
2.  Copy the `set_processor.sh` script separately and move it to the root folder.